### PR TITLE
support for Rails 6.0 and Ruby 3.0

### PR DESCRIPTION
**Pull request by ashkulz**

### CODE REVIEW

**Comment by ashkulz** [comment]
@urkle can you please review this? We use this gem :slightly_smiling_face:

File: spec/migration_spec.rb [hunk 1]
```diff
@@ -6,22 +6,22 @@
   before(:each) do
     define_schema do
 
-      create_table :users do |t|
-        t.string :login, :index => { :unique => true }
+      create_table :users, id: :integer do |t|
```
Comment (urkle) [inline spec/migration_spec.rb]: Instead of doing this, change the column types of the FKs to bigint.

**Review by urkle — CHANGES_REQUESTED** [review]
@ashkulz only one change (well, many places  changing integer -> bigint)

File: spec/migration_spec.rb (hunk 1 above)
Comment (ashkulz) [reply spec/migration_spec.rb]: That leads to a much bigger diff, as you need to change the type to `:bigint` everywhere -- see teamsimplepay/schema_auto_foreign_keys@9601958fbf73a12d5019f85cc50aff1ae439f660. I prefer the current approach, it's explicit and unlikely to break on Rails upgrades.

Do you want me to include the commit in this PR?

Comment (urkle) [reply spec/migration_spec.rb]: I'd rather include the bigtint change (as this is what I am doing with all the other gems)

Comment (urkle) [reply spec/migration_spec.rb]: And once this is in, we can create a new PR to get it supporting AR 6.1 & 7 (I have PRs for most of the other gems mostly done)

Comment (ashkulz) [reply spec/migration_spec.rb]: Pushed the changes, as requested :+1:

**Review by urkle — APPROVED** [review]

**Comment by urkle** [comment]
@ashkulz this is released now.   
I also have PR #6 setup to handle AR 6.1  & 7

**Pull request merged**